changelogs/fragments/49-k8s-loop-flattening-and-continue_on_error.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - k8s - Handle list of definition for option `template`. `continue_on_error` option added (whether to continue on creation/deletion errors) (https://github.com/ansible-collections/kubernetes.core/pull/49).
```

Comment:
Could you split these two sentences out into separate list items? These are two separate, but related, new features.

Reply:
@gravesm will do it. thanks